src/App.tsx

```diff
@@ -15,48 +21,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortTypeString | string,
+  isReversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortTypeString.Alphabet:
+          return good1.localeCompare(good2);
+
+        case SortTypeString.Length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReversed) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+  const visibleGoods = getPreparedGoods(goodsFromServer, sortField, isReversed);
+  const isResetVisible = sortField || isReversed;
+  const handleReset = () => {
```

Comment:
```suggestion
  const isResetVisible = sortField || isReversed;

  const handleReset = () => {
```

add space between function